### PR TITLE
Cartrefactor

### PR DESCRIPTION
**Pull request by sergeysila**
Fix login and cart tests

### CODE REVIEW

File: Automation.Common/Pages/CheckOutShippingPage.cs [hunk 1]
```diff
@@ -33,17 +38,31 @@ public class CheckOutShippingPage : BasePage
         [FindsBy(How = How.XPath, Using = "//button[@class='btn btn-order']")]
         private IWebElement _continueButton;
 
+        public void ChooseAddress()
+        {
+            if (_allAddress.Count > 1)
```
Comment (ViacheslavMelnichenko) [inline Automation.Common/Pages/CheckOutShippingPage.cs]: switch to _allAddress.Any()

File: Automation.Tests/app.config [hunk 2]
```diff
@@ -5,7 +5,7 @@
   </configSections>
     <appSettings>
       <add key="browser" value="chrome" />
-      <add key="homeUrl" value="http://10.17.194.115/" />
+      <add key="homeUrl" value="https://5c2d5468.ngrok.io/" />
```
Comment (ViacheslavMelnichenko) [inline Automation.Tests/app.config]: do not commit this!

Comment (sergeysila) [reply Automation.Tests/app.config]: What should we commit?

Comment (ViacheslavMelnichenko) [reply Automation.Tests/app.config]: pf, remove this changes!

**Pull request merged**